package.json

```diff
@@ -315,6 +315,7 @@
     "fs-extra": "8.1.0",
     "identity-obj-proxy": "3.0.0",
     "jest": "26.6.3",
+    "jest-junit": "13.0.0",
```

Comment:
The security review is labeled `blocked/waiting-for-changes`, so this can't be merged yet anyway.
Once the audit findings are fixed in jest-junit, we'll just pin to the next version.

Reply:
that doesn't seem to be coming soon so let's revert this to be greater than till they release a new version

Reply:
Done.